src/Maestro/Maestro.MergePolicies/DontAutomergeDowngradesMergePolicy.cs

```diff
@@ -19,15 +19,22 @@ public override Task<MergePolicyEvaluationResult> EvaluateAsync(IPullRequest pr,
         {
             try
             {
-                if (HasAnyDowngrade(pr))
+                List<string> versionCheckMessages = GetDowngradeOrInvalidVersionMessages(pr);
+
+                if (versionCheckMessages.Count > 0)
                 {
-                    return Task.FromResult(
-                        Fail("Some dependency updates are downgrades. Aborting auto-merge."));
+                    // It'd be great if this could be markdown, but checks as implemented today are just a big run-on sentence with no special formatting characters.
+                    string errorMessage = @$"
+The following dependency updates appear to be downgrades or invalid versions: {string.Join(',', versionCheckMessages)}. Aborting auto-merge.
+ Note that this may be caused by manual commits pushed to fix up the pull request (does not re-evaluate if you manually correct the pull request, though this is still acceptable to do).
```

Comment:
I feel something like this better explains that there's nothing to do for the user:

```suggestion
 Note that manual commits pushed to fix up the pull request won't cause the downgrade check to be re-evaluated, 
 you can ignore the check in this case.
```